core/types/rollup_cost.go

```diff
@@ -387,6 +387,12 @@ func EstimatedL1SizeScaled(costData RollupCostData) *big.Int {
 	return estimatedSize
 }
 
+// EstimatedDASize is the unscalled version of EstimatedDASizeScaled.
+func EstimatedDASize(costData RollupCostData) *big.Int {
```

Comment:
thoughts on making this a method on `*RollupCostData` rather than accepting it as a parameter?

Reply:
i like it, a bit more aesthetic. changed.